src/components/builds/ConfigurationWithIssues.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { createStyles, Theme } from '@material-ui/core';
+import { WithStyles, withStyles } from '@material-ui/core/styles';
+import { Alert } from '@material-ui/lab';
+import { createFragmentContainer } from 'react-relay';
+import { graphql } from 'babel-plugin-relay/macro';
+import { ConfigurationWithIssues_build } from './__generated__/ConfigurationWithIssues_build.graphql';
+import Accordion from '@material-ui/core/Accordion';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import Typography from '@material-ui/core/Typography';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    configurationTable: {
+      color: theme.palette.type === 'dark' ? theme.palette.primary.light : theme.palette.primary.dark,
+      background: theme.palette.type === 'dark' ? theme.palette.primary.dark : theme.palette.primary.light,
+      width: '100%',
+      borderSpacing: '0',
+    },
+    lineNumber: {
+      // An attempt to be consistent in colors with chips[1] backgrounds used on the same builds page
+      //
+      // [1]: https://github.com/mui-org/material-ui/blob/bda3d8541b2f3bea14ed241ae8155239d524a24c/packages/material-ui/src/Chip/Chip.js#L14
+      background: theme.palette.type === 'dark' ? theme.palette.grey[700] : theme.palette.grey[300],
+      fontFamily: 'Monaco, monospace',
+      paddingRight: theme.spacing(1),
+      width: '1%',
+      minWidth: theme.spacing(5),
+      textAlign: 'right',
+    },
+    lineContent: {
+      fontFamily: 'Monaco, monospace',
+      paddingLeft: theme.spacing(1),
+      whiteSpace: 'pre-wrap',
+    },
+    issue: {
+      fontSize: theme.typography.fontSize,
+      borderRadius: '0',
+    },
+  });
+
+interface Issue {
+  level: string;
+  message: string;
+  line: number;
+  column: number;
+}
```

Comment:
Seems it's no longer used

Reply:
Removed in https://github.com/cirruslabs/cirrus-ci-web/pull/351/commits/2586def22213cd4e6c00bdfa84e4690ee5b4350f.